pkg/controllers/securitypolicy_controller.go

```diff
@@ -41,48 +44,66 @@ type SecurityPolicyReconciler struct {
 	Service *services.SecurityPolicyService
 }
 
+func (r *SecurityPolicyReconciler) counter_inc(counter *prometheus.CounterVec) {
```

Comment:
Could we put the counter_inc func into metrics.py? If we add  more controllers in future, we can use them to update metrics value.

Reply:
done